seatunnel-connectors/seatunnel-connectors-spark/seatunnel-connector-spark-redis/src/main/scala/org/apache/seatunnel/spark/sink/Redis.scala

```diff
@@ -38,7 +38,7 @@ class Redis extends SparkBatchSink with Logging {
     val redisConfigs = new RedisConfig(RedisEndpoint(
       host = config.getString(HOST),
       port = config.getInt(PORT),
-      auth = config.getString(AUTH),
+      auth = if (config.getIsNull(AUTH)) null else config.getString(AUTH),
```

Comment:
Already put it into DefaultConfig in line 75, So `AUTH` alway be not null, same as `DATA_TYPE`

Reply:
`DEFAULT_AUTH` is null, `config.getString(DEFAULT_AUTH)` can get an error

Reply:
OK, I didn't notice.